src/data/markdown/docs/03 cloud/05 Project and Team Management/04 SAML SSO.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'SAML SSO'
+excerpt: 'Guide on setting up a SAML SSO Identity Provider with k6 Cloud'
+---
+
+> ⭐️ SAML SSO is available on Enterprise plans.
+
+## What is SAML?
+
+Security Assertion Markup Language (SAML) is an open standard for exchanging authentication and authorization data between parties, in particular, between an Identity Provider (e.g. Azure AD) and a Service Provider (e.g. k6 Cloud). The SAML SSO eases provisioning, management and access of users from a single identity provider, e.g. Okta, Azure AD or others, to multiple service providers.
+
+k6 Cloud makes use of SAML single sign-on (SSO) to facilitate provisioning of new users into the app and later giving them access to various organization(s) and projects, and in turn load tests and test runs.
+
+Read more over at Wikipedia about [Security Assertion Markup Language](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) and [SAML 2.0](https://en.wikipedia.org/wiki/SAML_2.0).
+
+## IdP and SP setup
+
+To setup SAML SSO for your organization, some requirements should be satisfied and some steps should be taken on your IdP and on k6 Cloud. Assuming you already have admin access to your IdP and the k6 Cloud, this is the usual process to enable SAML SSO for your organization on k6 Cloud:
+
+1. Install or setup the SAML SSO application on your IdP.
+2. Provide the "IdP Metadata URL" to the customer success team.
+3. Update the application on the IdP with the provided token from the customer success team.
+4. Assign users to the IdP application.
+5. Optionally choose default projects for newly provisioned users to land in on k6 Cloud.
+
+## Supported IdPs
```

Comment:
wasn't it mentioned at some point that basically any provider would work as long as they use SAML 2.0 standard? If that is so, perhaps we can add a line here somewhere to reflect that?